### ISSUE TITLE
Replace `remove_empty_values: false` in `2.0.0.pre2`

### ISSUE THREAD

**Issue by YassineDM** [tilo/smarter_csv#213]
Hi there,

I'm trying to switch to v2 with its new options but I'm not sure how to proceed.

Previously, I was just setting `remove_empty_values: false`. Now I see that it may have been replaced with `hash_transformations: [:remove_blank_values ]`. But even setting that to an empty array doesn't give the same result...

Anyone could help?

**Comment by tilo** [comment]
Hi Yassine!

do you have a sample CSV file?

**Comment by tilo** [comment]
@YassineDM ?

**Comment by YassineDM** [comment]
@tilo,

Before trying it with a CSV file, my question was about the new DSL for options, which are currently:
```ruby
csv = # csv_file
csv_options = {
    chunk_size: 10_000,
    file_encoding: 'utf-8',
    col_sep: ',', quote_char: '"',
    headers_in_file: true,
    convert_values_to_numeric: false,
    remove_empty_values: false
  }

SmarterCSV.process(csv, **csv_options)
```

How would we write equivalent `csv_options`?

**Comment by tilo** [comment]
This is ear-marked to be addressed in v2.0
